### DISCUSSION TITLE
How to get event when modal is closed..??

### DISCUSSION THREAD

**Discussion by asenseofpradhyu** [chakra-ui/chakra-ui#4820] [Q&A]
How to get event when modal is closed..?? Like when i click close button then some code should run..

For example:- When i close modal textarea should clear and state should be clear..

Any Idea Developers ?

**Comment by asobirov** [comment]
```
const { isOpen, onOpen, onClose } = useDisclosure()
const someFunc = () => {
   console.log('henlo')
   onClose()
}
 <Modal onClose={someFunc}>
 ```

Or something similar to this should work
